_pages/en_US/basic-information.txt

```diff
@@ -0,0 +1,76 @@
+
+## Downloading, copying, and extracting files
+
+![]({{ "/images/screenshots/onboarding/github.png" | absolute_url }})
+{: .notice--info}
+
+A large part of this guide involves copying files that are downloaded on your computer/phone to your 3DS SD card. Many of these files come from GitHub, so refer to the above screenshot if you're unfamiliar with how to download files from there.
+
+![]({{ "/images/screenshots/onboarding/zipfiles.png" | absolute_url }})
+{: .notice--info}
+
+Many of the files that you will be asked to download come in the `.zip` file format. Doing this allows multiple files to be downloaded at once, which makes it easier to organize what is being downloaded.
+
+When you are told to "copy (file) from (file).zip", that means that you need to open the ZIP file. Inside the ZIP file will be the file you're looking for, from which you can drag-and-drop it to the specified location on your SD card.
+
+On most operating systems, you can extract files by simply double-clicking the file. If this doesn't work for whatever reason, you can install a reputable third-party extraction tool like [7-zip](https://7-zip.org/) instead.
+
+## Windows only: File extensions
+
+![]({{ "/images/screenshots/onboarding/fileext.png" | absolute_url }})
+{: .notice--info}
+
+If your computer is running Windows, file extensions may be hidden by default. Depending on the applications installed on your computer, this may make it harder to find the files you're looking for. To enable file extensions, follow [these instructions](file-extensions-(windows)).
+
+## SD card location and physical size
+
+![]({{ "/images/screenshots/onboarding/sdcard.jpg" | absolute_url }})
+{: .notice--info}
+
+The location of the SD card slot and the type of SD card that is accepted depends on the model of your 3DS. More information can be found on [Nintendo's official support page](https://en-americas-support.nintendo.com/app/answers/detail/a_id/271/~/how-to-insert-an-sd-card-or-microsd-card).
+
+- On the **Old 3DS**, **Old 3DS XL**, and **2DS**, the SD card is hidden behind a flap on the left or right side of the console. These consoles use a **large-size SD card**. A **microSD card in an SD adapter** will also work.
+- On the **New 3DS** and **New 3DS XL**, the SD card is behind the backplate, which you will need a screwdriver to remove. These consoles use a **microSD card**.
+- On the **New 2DS XL**, the SD card is under the same flap as the cartridge slot. This console uses a **microSD card**.
+
+## SD card capacity
+
+Despite Nintendo's support page, the 3DS can read SD cards larger than 32GB. The main constraint is that the 3DS can only read SD cards formatted as FAT32 (**not** exFAT or NTFS). You can use one of the following tools to format your SD card, depending on your operating system: [Windows](formatting-sd-(windows)), [macOS](formatting-sd-(mac)), [Linux](formatting-sd-(linux)).
```

Comment:
Can probably remove "Despite Nintendo's support page" and switch it to "The 3DS can read SD cards larger than 32GB, but 128GB is the maximum before you can start running into technical problems" or something?

Reply:
i vote no. there is a surprising amount of people who when told 64gb/128gb etc sd cards work, reply 'but nintendo said?'

Reply:
The current reword says "The 3DS can read SD cards of any capacity (even those larger than 32GB)". I'm thinking tacking on "despite Nintendo's claims" after 32GB, but it seems a bit awkward. Don't know if "even those larger than 32GB" says implicitly enough that people normally believe that 32GB cards don't work, but that they actually do.